components/brave_wallet/browser/brave_wallet_constants.h

```diff
@@ -622,6 +622,8 @@ const char kAffiliateAddress[] = "0xbd9420A98a7Bd6B89765e5715e169481602D9c3d";
 
 const int64_t kBlockTrackerDefaultTimeInSeconds = 20;
 
+const char kPolygonMainnetEndpoint[] = "https://mainnet-polygon.brave.com/";
```

Comment:
constexpr

Reply:
added many of them
https://github.com/brave/brave-core/pull/13202/commits/7d207e078107633244e3172548633c6c7187aff1